assets/js/components/module/ModuleApp.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * ModuleApp component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { _x, sprintf, __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { isFeatureEnabled } from '../../features';
+import Header from '../Header';
+import Alert from '../Alert';
+import PageHeader from '../PageHeader';
+import ModuleFooter from './ModuleFooter';
+import LegacyModuleApp from './LegacyModuleApp';
+import WidgetContextRenderer from '../../googlesitekit/widgets/components/WidgetContextRenderer';
+
+import DateRangeSelector from '../DateRangeSelector';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+
+const { useSelect } = Data;
+
+function ModuleApp( { moduleSlug } ) {
+	const screenWidgetContext = useSelect( ( select ) => select( CORE_MODULES ).getScreenWidgetContext( moduleSlug ) );
+	const { slug, name } = moduleSlug;
+	const serviceName = _x( name, 'Service name', 'google-site-kit' ); // eslint-disable-line @wordpress/i18n-no-variables
```

Comment:
Why is this rule disabled? It seems the violation here is legitimate and this won't be able to be extracted by the i18n tools: https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/i18n-no-variables.md

Reply:
Removed as discussed this morning - thanks 👍